schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,222 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Network",
+    "description": "Microsoft Network Resource Types",
+    "resourceDefinitions": {
+        "trafficManagerProfiles": {
```

Comment:
We unified all JSON files to 2 space indentation after this PR was requested.
Vlad, please help to reformat to 2 space indentation for this file. Thanks!